### ISSUE TITLE
GUI crash macOS Monterey

### ISSUE THREAD

**Issue by The-Big-V** [samschott/maestral#514]
**Describe the bug**
Since upgrading to macOS 12.0.1 Monterey, Maestral GUI is crashing.

Upon launching Maestral.app I get two crash reports:

1. From Maestral.app
2. From nsattributedstringagent

The Maestral process continues running  and continues syncing with Dropbox, and I continue getting macOS notifications from Maestral. However, there is no Maestral item in the menu bar.

**System:**
 - Maestral version: 1.5.1 (have tried both macOS bundle and brew install)
 - Python version: 3.9.7
 - OS: macOS 12.0.1

**Additional context**

Crash reports:

- [maestral-crash.txt](https://github.com/SamSchott/maestral/files/7473975/maestral-crash.txt)
- [nsattributedstringagent-crash.txt](https://github.com/SamSchott/maestral/files/7473976/nsattributedstringagent-crash.txt)

Is anyone else experiencing the same problem?

**Comment by SamSchott** [comment]
Thanks for the report! Does the crash happen each time or can you sometimes successfully launch the GUI?Just to confirm, you are on an Intel Mac, right? I'm on macOS 12.0.1 myself and am not seeing any problems. The simultaneous crash of the `nsattributedstringagent` might be an indication of what is going on but I'll need to look into it.

The sync process will continue to run in the background in case of a hard segfault of the GUI. This is of course not ideal but any solutions to this would be very platform-specific and haven't gotten around to implement such guards yet.

**Comment by The-Big-V** [comment]
On initial launch the GUI will allow me to enter an API token and select which folders to sync. But then the app crashes and it doesn't appear in the menu bar, (then the GUI crashes on any subsequent attempt to launch).

Also yes, it's an intel Mac.

Since everything is syncing, and I can access every function from CLI, it doesn't seem like there's any urgent need for a fix. Whenever macOS 12.1 is released, I'll report back as to whether the issue persists.

**Comment by SamSchott** [comment]
> On initial launch the GUI will allow me to enter an API token and select which folders to sync. But then the app crashes and it doesn't appear in the menu bar, (then the GUI crashes on any subsequent attempt to launch).

Could you post any crash report that you get from macOS? This might give some insight on the exact native API call which triggers the crash, if we are lucky.

**Comment by applecodingacademy** [comment]
With M1 Mac mini app doesn't work and only shows: Could not access sys._traceback.
Monterey 12.0.1

**Comment by frozenpandaman** [comment]
I'm not sure why, but I'm also on Monterey 12.0.1 on a M1 machine and running it successfully. I have gotten crashes in the past (sometimes two instances try to start up) but force quitting & restarting everything has always seemed to fix it.

**Comment by The-Big-V** [comment]
> Could you post any crash report that you get from macOS?

Did you note the crash reports linked in the original post?

**Comment by SamSchott** [comment]
@The-Big-V, apologies, you of course provided them already! Unfortunately, the crash report only contains trackbacks on the libffi level (used to bridge Python and Objective-C). This, together with not being able to reproduce the crash on my machine, makes it quite difficult to debug.

Since the NSAttributetString agent crashes at the same time, my best bet is that issue is related to the use of those. I'll check the relevant parts of the code.

**Comment by cweirup** [comment]
I'm also experiencing the same issue, currently on Monterey 12.1. The other thing I see is the Maestral icon when I Command-Tab between apps, but if I select Maestral, nothing happens in the UI.

I can send crash reports if someone can point me to where they are getting generated.

**Comment by The-Big-V** [comment]
Hi @cweirup , just open your "Applications>Utilities>Console.app," and in the side pane there's a section for crash reports; look for one labelled as "Maestral."

**Comment by The-Big-V** [comment]
I just updated to Maestral 1.5.3

Now, when I enter "Maestral GUI" from terminal, I still get the crash as described above, but I also get the following message in terminal:

`/usr/local/bin/maestral: line 12:  6448 Segmentation fault: 11  "$DIR/Maestral" --cli "$@"`

**Comment by ChrisChinchilla** [comment]
I am not sure if this is related, but I was also getting the "Could not access sys._traceback" error when launching the GUI suddenly, but the CLI worked. Then I finally realised today that it was because I had manually installed the Sparkle Framework into my library/frameworks folder while experimenting with a Qt project and deleting that caused the Maestral GUI to work again.

I am not sure if this is what causes others the issue, but you never know!
